package-lock.json

```diff
@@ -507,9 +507,9 @@
       "dev": true
     },
     "@blackbaud/help-client": {
-      "version": "2.0.2",
-      "resolved": "https://registry.npmjs.org/@blackbaud/help-client/-/help-client-2.0.2.tgz",
-      "integrity": "sha512-C0JezDxjm6zGcS5jyD6xXHxQ8xMxMva6ztDkKJv9pFs+NmR9xQ3evn+Mpl+yBjKAXCJFMDXJVudk6gE466osuw==",
+      "version": "2.1.0",
+      "resolved": "https://blackbaud.pkgs.visualstudio.com/_packaging/Blackbaud-npm/npm/registry/@blackbaud/help-client/-/help-client-2.1.0.tgz",
```

Comment:
it's not clear to me that this is a good idea. it seems not to be. but i also dont know how to make my system support both.

Reply:
Agreed it's not a good idea.  Let's see if we can work together to resolve it offline.